extension_service/example-config.yml

```diff
@@ -3,8 +3,8 @@ host: 0.0.0.0
 datastore:
   # Example for AlloyDB
   kind: "postgres"
-  host: 127.0.0.1
-  # port: 5432
-  database: "my_database"
-  user: "my-user"
-  password: "my-password"
\ No newline at end of file
+  host: DB_HOST
```

Comment:
Does this hurt readability of this config? I wonder if we should just find/replace around it